packages/typespec-client-generator-core/lib/decorators.tsp

```diff
@@ -92,6 +92,8 @@ extern dec client(target: Namespace | Interface, value?: {}, scope?: valueof str
 extern dec operationGroup(target: Namespace | Interface, scope?: valueof string);
 
 /**
+ * DEPRECATED: Use `@usage` and `@access` decorator instead.
```

Comment:
Do you want to mark them explicitly as deprecated instead of just a comment that most people don't read?

Reply:
@timotheeguerin they've already been previously marked as deprecated. They've been deprecated for a while, is it fair for us to remove support for it now?

Reply:
we have still never removed any deprecated element from typespec but for tcgc probably ok if you do as it shouldn't be used in RPSaaSmaster yet and otherwise it would have already failed